### ISSUE TITLE
seek solution for www.bildindex.de

### ISSUE THREAD

**Issue by dancorpse** [lovasoa/dezoomify#75]
http://www.bildindex.de/document/obj00174541/fmc654619, this is a zoomify image, but the following url does not work in dezoomify, http://www.bildindex.de/media/zoomify/fmc654619/ImageProperties.xml; Am I anything wrong?
 I notice that there're very long stuffs begin with "?Expires=" after "ImagePropertiex.xml, Is there any thing to do with? (I use dezoomify via proxy) But dezoomify works fine with other sites.

**Comment by lovasoa** [comment]
Looks like they use the same system as #79 . You have to first access the tile `http://www.bildindex.de/media/proxy/67DA78F7E4C940A8B37729BBA513A71B/TileGroup0/4-3-1.jpg` with the right `Referer` and `Cookie` headers, and then receive a redirection to a tile URL that is valid only for a short period of time.

**Comment by gregg128** [comment]
Any progress on this site? 
Thanks!

**Comment by leechers** [comment]
Well, if you want the max resolution, there is a way. Use Mozilla Firefox, find the image , press the zoom button, do CTRL-SHIFT-M, enter the max resolution 9999x9999. Wait for the image to be fully loaded (scroll up and down, left and right). When fully loaded, press the 'camera' button in the upper screen (find it! by scrolling left and right) which will take a screenshot of the photo. Now you have the maximized fully zoom image from the screen in your Downloads directory. It's not as easy as dezoomify, but it is not very hard to get the full resolution image.

**Comment by tomybyte** [comment]
@leechers excellent solution!

**Comment by lovasoa** [comment]
If someone is interested in finally making dezoomify work with bildindex.de, here is what I found out:

 - the tiles are requested through a proxy
 - The url to the proxy has to be found dynamically: https://www.bildindex.de/db/apsisa.dll/eteRessourceResolver?mediumID=fmc654619&mediumType=zoom&doctype=obj&docnumber=00174541&_=1578336598735
 - Once the proxy URL is found, resources have to be requested at the proxy URL, while taking care of setting all cookies correctly.

**Comment by tomybyte** [comment]
@lovasoa what can I do in dezoomify with the www.bildindex.de/eteRessourceResolver URL ?

**Comment by lovasoa** [comment]
Currently, nothing. I was just posting a message with my latest findings in case someone is interested. This information is going to be required if one wants to one day make dezoomify work with that site.

**Comment by tomybyte** [comment]
@lovasoa thanks for clarifying!

**Comment by lovasoa** [comment]
Ok, I managed to get it working with [dezoomify-rs](https://github.com/lovasoa/dezoomify-rs), but it's still a little convoluted:

1. While browsing an image on bildindex.de, open the network inspector of your browser, and find a request to `eteRessourceResolver` 
<img width="1285" alt="image" src="https://user-images.githubusercontent.com/552629/71973927-7d396380-3210-11ea-85a6-646d8403137e.png">

2. Copy the url from the response. In this case: ` https://www.bildindex.de/media/proxy/F8F639869A7A4DF6A33F9CAA138D1B2D/ `

3. In the cookies tab, find the sid cookie. In our case: `{2eb572ad-5e26-463d-a12e-739591583da7}`
<img width="1675" alt="image" src="https://user-images.githubusercontent.com/552629/71974029-c12c6880-3210-11ea-85c2-399606178688.png">
4. In a terminal, launch dezoomify-rs with the following parameters, using the information found during the previous steps:

```sh
./dezoomify-rs --header 'Cookie: sid={2eb572ad-5e26-463d-a12e-739591583da7}' 'https://www.bildindex.de/media/proxy/F8F639869A7A4DF6A33F9CAA138D1B2D/ImageProperties.xml'
```

**Comment by lovasoa** [comment]
![dezoomified](https://user-images.githubusercontent.com/552629/71974259-5d566f80-3211-11ea-9ea5-a375966d00a1.jpg)

**Comment by tomybyte** [comment]
@lovasoa this works not for me :(
`d:\_Proggys\dezoomify-rs 1.6.0>dezoomify-rs --header 'Cookie: sid={32704313-9237-429e-a172-723d9fd2fe71}' 'https://www.bildindex.de/media/proxy/DEF9DD97793347D695E41E20C1206301/ImageProperties.xml'
Trying to locate a zoomable image...
Opening sid={32704313-9237-429e-a172-723d9fd2fe71}'...
Input/Output error: Das System kann die angegebene Datei nicht finden. (os error 2)`

**Comment by lovasoa** [comment]
Oh, I always forget that the string delimiters are not the same in windows and MacOS :

```
dezoomify-rs --header "Cookie: sid={32704313-9237-429e-a172-723d9fd2fe71}" "https://www.bildindex.de/media/proxy/DEF9DD97793347D695E41E20C1206301/ImageProperties.xml"
```

should work in both windows and macos

**Comment by tomybyte** [comment]
@lovasoa great job, thanks a lot!

**Comment by mcn-ops** [comment]
Is this still working? It doesn't seem to be for me.

**Comment by lovasoa** [comment]
Yes, it does still work. Did you extract the right cookie from your browser ?

**Comment by mcn-ops** [comment]
I'm sure I did. I also used the link you provided and got the same error. When I click the Dezoomify button, it says 'Please enter a URL'.

**Comment by portraitsofrulers** [comment]
Hello! 
I'm trying to download this image using dezoomify-rs: `https://www.bildindex.de/media/obj00011523/fmd10002408?medium=fmd10002408`
 
According to your instructions, I received the command: `./dezoomify-rs --header "Cookie: sid={938f56e3-9458-4026-a103-bfbd11f54f19}" "https://www.bildindex.de/media/proxy/57D7E3A954A241AB8563D93C88714C3D/ImageProperties.xml"`
 
But when I try to run the program, an error is thrown that it is impossible to determine the image format. 

What to do to get the full size image?

![screen1](https://user-images.githubusercontent.com/61700988/148651111-c6f919fe-bc9e-4ed9-8e03-f01a15d5c249.jpg)  
![screen2](https://user-images.githubusercontent.com/61700988/148651126-04e655e6-57f1-49ed-b481-a61b192a94de.jpg)
![screen3](https://user-images.githubusercontent.com/61700988/148651131-7c15263b-1728-4811-9c44-28ffe33e9b24.jpg)

**Comment by jakovgan** [comment]
Also does not work for me. I double checked everything 5 times. Always get the same answer: 

ERROR Dezoomer error: Unable to create the dezoomer: Tried all of the dezoomers, none succeeded. They returned the following errors:

 - custom: The 'custom' dezoomer cannot handle this URI
 - google_arts_and_culture: The 'google_arts_and_culture' dezoomer cannot handle this URI
 - generic: The 'generic' dezoomer cannot handle this URI
 - pff: The 'pff' dezoomer cannot handle this URI
 - IIPImage: The 'IIPImage' dezoomer cannot handle this URI
 - nypl: The 'nypl' dezoomer cannot handle this URI
 - zoomify: Unable to download required data: Input/Output error: No such file or directory (os error 2)
 - iiif: Unable to download required data: Input/Output error: No such file or directory (os error 2)
 - deepzoom: Unable to download required data: Input/Output error: No such file or directory (os error 2)
 - krpano: Unable to download required data: Input/Output error: No such file or directory (os error 2)

dezoomify-rs expects a zoomable image meta-information file URL. To find this URL, you can use the dezoomify browser extension, which you can download at
- https://lovasoa.github.io/dezoomify-extension/ 
If this doesn't help, then your image may be in a format that is not yet supported by dezoomify-rs.
You can ask for a new format to be supported by opening a new issue on https://github.com/lovasoa/dezoomify-rs/issues

Enter an URL or a path to a tiles.yaml file:

**Comment by jakovgan** [comment]
It worked now, you have to run the terminal and not use dezoomify-rs directly

**Comment by rudolphos** [comment]
@lovasoa 

This worked. Thanks

![Image](https://github.com/user-attachments/assets/2644bde3-7278-489b-a04b-eb57ea2000e4)

**Comment by maxuju1** [comment]
Hi! I can't seem to find the correct URL that I need to copy. I saw it always included the terms "media/proxy" so I looked for it directly but it didn't show up. In fact, nothing ever shows up with the type "xhr" which it seems to be.

<img width="1440" alt="Image" src="https://github.com/user-attachments/assets/b5640c2b-8104-4ed0-a1bd-efa27c144064" />

**Comment by otofoto** [comment]
Just for notice if only interested in some photos

_Please note our new price list, according to which the provision of
digital copies in high resolution quality of less than 10 copies is free
of charge since 1 January 2025:_

https://www.uni-marburg.de/de/fotomarburg/service/preisliste

**Comment by lovasoa** [comment]
Bot-assisted investigation note: The refreshed corpus still shows the original Bildindex Zoomify metadata URL returning 403, while the object page exposes session-specific `media/proxy/.../` Zoomify/OpenLayers paths. This matches the existing thread: Bildindex requires a browser-derived proxy URL plus the `sid` cookie, and later comments confirm the documented dezoomify-rs terminal workflow works. I would keep this open as site-specific support/workaround documentation and use `solved` as the primary triage label rather than closing as a duplicate.

**Comment by lovasoa** [comment]
Bot-assisted triage note: closing this issue because it is currently labeled `solved`.
